strongback/docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Houston docker file
+# Builds an ubuntu base with liftoff for packaging debian applications
```

Comment:
Why exactly are we doing this? liftoff creates it's own chroot, so the filesystem is isolated, running it on it's own secured VM should be good enough once we let other people into houston